ports/descent3/Descent 3.sh

```diff
@@ -56,6 +56,7 @@ fi
 
 if [ "$LIBGL_FB" != "" ]; then
   export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es.$DEVICE_ARCH/libGL.so.1"
+  export SDL_VIDEO_EGL_DRIVER="$GAMEDIR/gl4es.$DEVICE_ARCH/libEGL.so.1"
```

Comment:
Use an if statement to check for Knulli instead so it only gets applied for Knulli
```suggestion
  if [[ "${CFW_NAME^^}" == "KNULLI" ]]; then
    export SDL_VIDEO_EGL_DRIVER="$GAMEDIR/gl4es.$DEVICE_ARCH/libEGL.so.1"
  fi
```

Reply:
Likewise, the presence of libEGL.so.1 has been enough to cause problems on other CFW.  It might be a good idea to have the script move or rename libEGL.so.1 to something else when not running Knulli perhaps.

Reply:
Why does Knulli need EGL in the first place, and if other cfw have it in their `usr/lib`, why doesn't Knulli?

Reply:
Sounds like Knulli needs to address this IMO